openapi.yaml

```diff
@@ -330,15 +330,15 @@ components:
         quotaMaxBytes:
           type: integer
           format: int64
-          description: "Maximum storage space used by the node"
+          description: "Maximum storage space (in bytes) used by the node on the hard drive."
```

Comment:
```suggestion
          description: "Maximum storage space (in bytes) available for the node in Codex's local repository."
```

Reply:
Yeah, I wasn't sure how far I could go with the rewording, which is why I kept  `used by the node`.  But your suggestion `available for the node in Codex's local repository` is much better.